client/landing/gutenboarding/onboarding-block/style.scss

```diff
@@ -35,3 +35,102 @@
 	text-decoration: none;
 	position: relative;
 }
+
+.onboarding-block__question input {
+	background: transparent;
+}
+
+.onboarding-block__background {
+	position: fixed;
+	top: 0;
+	bottom: 0;
+	width: 100%;
+	background-image: url( '/calypso/images/verticals/default.jpg' );
+	background-size: cover;
+	z-index: -1;
+}
+
+.onboarding-block__background::before {
+	background: $light-gray-100;
+	content: '';
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	opacity: 0.7;
+	transition: opacity 1s linear;
+}
+.onboarding-block__background-fade-enter {
+	&.onboarding-block__background::before {
+		opacity: 0.9;
+	}
+}
+.onboarding-block__background-fade-enter,
+.onboarding-block__background-fade-exit,
+.onboarding-block__background-fade-exit-done {
+	&.onboarding-block__background {
+		opacity: 0;
+	}
+}
+.onboarding-block__background-fade-exit,
+.onboarding-block__background-fade-enter-done {
+	transition: opacity 1s linear;
+}
+
+.onboarding-block__background[data-vertical^='p1'] {
+	background-image: url( '/calypso/images/verticals/p1.jpg' );
+}
+.onboarding-block__background[data-vertical^='p2'] {
+	background-image: url( '/calypso/images/verticals/p2.jpg' );
+}
+.onboarding-block__background[data-vertical='p2v6'] {
+	background-image: url( '/calypso/images/verticals/p2v6.jpg' );
+}
+.onboarding-block__background[data-vertical^='p13'] {
+	background-image: url( '/calypso/images/verticals/p13.jpg' );
+}
+.onboarding-block__background[data-vertical^='p31'],
+.onboarding-block__background[data-vertical^='p12v30'] {
+	background-image: url( '/calypso/images/verticals/p31.jpg' );
+}
+.onboarding-block__background[data-vertical^='p12v2'] {
+	background-image: url( '/calypso/images/verticals/p12v2.jpg' );
+}
+.onboarding-block__background[data-vertical^='p31v28'] {
+	background-image: url( '/calypso/images/verticals/p31v28.jpg' );
+}
+.onboarding-block__background[data-vertical^='p19'] {
+	background-image: url( '/calypso/images/verticals/p19.jpg' );
+}
+.onboarding-block__background[data-vertical^='p30'] {
+	background-image: url( '/calypso/images/verticals/p30.jpg' );
+}
+
+@media ( prefers-color-scheme: dark ) {
```

Comment:
Having a dark background might be more than just personal preference. It seems to be also an accessibility feature. And now this media query is supported by major browsers as well.

Learn more: https://web.dev/prefers-color-scheme/